arelle/WebCache.py

```diff
@@ -84,7 +89,11 @@ class WebCache:
 
     default_timeout = None
 
-    def __init__(self, cntlr, httpProxyTuple):
+    def __init__(
+        self, cntlr: Cntlr,
+        httpProxyTuple: Optional[tuple[bool, str, str, str, str]]
```

Comment:
```suggestion
        httpProxyTuple: tuple[bool, str, str, str, str] | None
```

Reply:
Also add `from future` etc in the import section

Reply:
> Also add `from future` etc in the import section

Yep got it

Reply:
done